package.json

```diff
@@ -142,6 +144,7 @@
     "jest-junit": "10.0.0",
     "lint-staged": "10.2.2",
     "prettier": "1.19.1",
+    "react-test-renderer": "16.12.0",
```

Comment:
where are we using this module? I don't see it imported into any our tests

Reply:
Its used by `react-hooks-testing-library` its a peer dependency that doesnt come bundled with i